conda/recipes/versions.yaml

```
@@ -37,7 +37,7 @@ boost_cpp_version:
 clang_version:
   - '=8.0.1'
 cmake_version:
-  - '=3.18'
+  - '=3.20.1'
```

Would suggest letting the patch version float (though we can set a lower limit if that's helpful)

```suggestion
  - '>=3.20.1,<3.21'
```